lavamoat/browserify/policy-override.json

```diff
@@ -1,121 +1,3 @@
 {
-  "resources": {
```

Comment:
Whoops - looks like you deleted all of the policy overrides. These aren't auto-generated, these need to be preserved.